smithy-typescript-codegen/src/main/java/software/amazon/smithy/typescript/codegen/integration/AddCompressionDependency.java

```diff
@@ -0,0 +1,149 @@
+/*
+ * Copyright 2023 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
nit: prefer using short-form header with no copyright year

Reply:
Remove Copyright year from header
```suggestion
 * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Checkstyle comment added in https://github.com/smithy-lang/smithy-typescript/pull/1131